client/engine/engine.go

```diff
@@ -343,7 +343,7 @@ func (e *Engine) attemptProgress(objective protocols.Objective) (outgoing Object
 	return
 }
 
-// SpawnConsensusChannelIfDirectFundObjective will attempt to create and store a ConsensusChannel derived from the supplied Objective iff it is a directfund.Objective.
```

Comment:
I think I wrote this, and I think maybe it wasn't a typo -- I may have meant "if and only if". I think your change here is good, regardless.

Reply:
I checked the authorship. If it had been Andrew I would have been **sure** that it wasn't a typo. Being you I settled on 50/50.

But in any case I think the "plain Enghlish" is better in places like this.